dotCMS/src/main/java/com/dotcms/exception/ExceptionUtil.java

```diff
@@ -0,0 +1,26 @@
+package com.dotcms.exception;
+
+/**
+ * Exception Utils
+ * @author andrecurione
+ */
+public class ExceptionUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)